.github/workflows/mvn-verify-check.yml

```diff
@@ -24,7 +24,32 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  mvn-verify-check:
+  package-aggregator:
+    strategy:
+      matrix:
+        spark-version: [312, 313, 320, 321, 322, 330]
```

Comment:
we have yet another place to update versions?  What is this list supposed to match?  why doesn't it have 321cdh?

Reply:
> we have yet another place to update versions? 

Unfortunately yes. There might be an elegant solution to generate this array in an upstream job, but I haven't looked at it yet. I would really like us to solve this once and for all including CI. 

> What is this list supposed to match? why doesn't it have 321cdh?

it's supposed to be like noSnapshots. I first had 321cdh too but then decided against a vendor shim in a github. I'll put it back for consistency. I don't have a particularly strong point for not including it.

Reply:
all the jars are available open source so I think should include it like normal build does